pulpcore/pulpcore/app/serializers/repository.py

```diff
@@ -266,6 +272,7 @@ class PublicationSerializer(ModelSerializer):
         parent_lookup_kwargs={'repository_pk': 'repository__pk'},
         read_only=True,
     )
+    # NOTE: models.Publication.complete field is being left out on purpose
```

Comment:
We don't specify other fields that are left out on other serializers, so I don't think we need to here, but its just my opinion. If you want to keep this, please explain why it is left out.